packages/nextjs/src/config/webpack.ts

```diff
@@ -5,6 +5,7 @@ import { default as SentryWebpackPlugin } from '@sentry/webpack-plugin';
 import * as chalk from 'chalk';
 import * as fs from 'fs';
 import * as path from 'path';
+import { WebpackPluginInstance } from 'webpack';
```

Comment:
m: Since we're only using it as a type, would it make sense to do `import type` here?

Reply:
Actually, it turns out I had to run it through `types.ts` because of https://github.com/pahen/madge/issues/306. (See the comment I left in the file.) But in any case, I changed the import from `types.ts` to be a type import.